docker-local-mojaloop/dfsp_a/secrets/dfsp_a_client.crt

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think we shouldn't keep certs in the repository... we should be able to generate them whenever we need them. Here is the example script: https://github.com/mojaloop/mojaloop-simulator/blob/master/scripts/createSecrets.sh

Reply:
I tried the script I think there is some additional stuff you need to do after creating secrets.

while we can generate them is there a major concern to keeping test keys and certs in the repo so a person can test it out quickly?

Reply:
You are right. If the real need to secure these keys appear we will act.